test/functional/legacy/045_folding_spec.lua

```diff
@@ -1,30 +1,34 @@
 -- Tests for folding.
+local Screen = require('test.functional.ui.screen')
 
 local helpers = require('test.functional.helpers')(after_each)
-local feed, insert, clear, execute, expect =
-  helpers.feed, helpers.insert, helpers.clear, helpers.execute, helpers.expect
+local feed, insert, execute, expect =
+  helpers.feed, helpers.insert, helpers.execute, helpers.expect
 
-describe('folding', function()
-  before_each(clear)
+describe('Folding ', function()
```

Comment:
We don't capitalize any test descriptions. Consistency is more important than whatever provoked this change.